docs-standards/10-structure-the-content/01-document-types/quickstart-help-me-get-started.md

```diff
@@ -0,0 +1,182 @@
+
+> ℹ️ **Note:** 
+>
+> For additional tips on writing a procedure, see [Procedure docs standards](procedure-help-me-to-do.md
+).
+
+#### Setup heading:
+
+Guidelines:
+
+- Use imperative verbs to describe the section's purpose: *Install*, *Configure*, *Set up* and so on.
+- Use H2 headings for each setup requirement.
+
+> ⚙️ **Example:**
+> 
+> - *Build an nwaku node*
+> - *Sign up for GitHub Copilot*
+
+### Task section
+
+Use this section to describe what users need to do. 
+
+Guidelines:
+
+- Choose two or three tasks that are essential, quick to complete, and provide immediate value to the user. 
+- If your quickstart involves a complex task, break it down into different logical subtasks with each subtask consisting of one or more related steps. 
+
+#### Task heading
+
+Guidelines:
+
+- Focus on the result, not on the task.
+- Start the title with an action verb in the imperative form. Don't use the -ing form of the verb.
+- Use H2 headings for each task.
+- Use H3 headings for each subtask.
+
+#### Task steps
+
+Guidelines:
+
+- Optionally, include an introduction paragraph to provide context or required knowledge for the task.
+- Include a short description for each step, even when it contains a code sample.
+- Provide examples of sample output, such as return data, a message, so that the users can validate whether they perform the step correctly or not.
+- Include one action in a step.
+- Limit the procedure to a maximum of seven steps. If you need more steps to explain the task, create a subtask.
+
+> ⚙️ **Example:**
+>
+> ## Manage files using Codex Vault
+>
+> You can use Codex Vault, a GUI web application, to manage your files on the Codex testnet. Once you have your Codex node running using the installer, you can access the Codex Vault at https://app.codex.storage.
+> ### Upload files
+>
+> 1. Open Codex Vault in your browser.
+> 1. In the Upload section, drag and drop your file or click **Upload** to choose it.
+> 1. Back up the file CID for download.
+>
+> ### Download files
+> 1. Open Codex Vault in your browser.
+> 1. In the Download section, enter the CID of the file you want to download.
+> 1. Click **Download**.
+
+> ℹ️ **Note:**
+>
+> For additional tips on writing tasks, see [Procedure docs standards](procedure-help-me-to-do.md
+).
+
+#### Code sample or image (optional)
+
+This section is part of the task steps and doesn't use a heading. 
+
+Use one image or code sample per step. If you need two, the step needs splitting. Make sure your code samples and images are up-to-date, functional, and relevant to the task. For more tips, check out [code](../../20-style-the-content/13-code.md) and [images](../../30-work-with-media/02-images.md) guidelines.
+
+> ℹ️ **Note:**
+>
+> When you use a code sample or image, it should be indented under the step description so that it's visually grouped with that step.
+
```

Comment:
In your table, you wrote `subtasks` between `code sample or image` and `next steps`, but I don't see `next steps` here.

<img width="1392" height="700" alt="Image" src="https://github.com/user-attachments/assets/dfcca29e-3b2e-4f57-9003-dd9bacf3b37d" />